dashboard/app/coverage_batch.go

```diff
@@ -25,10 +25,7 @@ func initCoverageBatches() {
 	http.HandleFunc("/cron/batch_coverage", handleBatchCoverage)
 }
 
-const (
-	daysToMerge         = 7
-	batchTimeoutSeconds = 60 * 60 * 6
-)
+const batchTimeoutSeconds = 60 * 60 * 8
```

Comment:
If 6 happened already, 8 likely to happen as well.
I would set it to 2-5x of the time we know happens. There is no particular penalty/downside of setting it to higher value (besides avoiding future false positives)..

Reply:
Ack. Changed to 12.